website/package.json

```
@@ -63,6 +63,7 @@
     "@types/react-router-dom": "5.3.3",
     "@types/react-scrollspy": "3.3.9",
     "@types/redux-mock-store": "1.5.0",
+    "@types/redux-state-sync": "^3.1.10",
```

```suggestion
    "@types/redux-state-sync": "3.1.10",
```

We use renovate to update our dependencies (though we haven't been very diligent in doing so)